grails-app/services/au/org/ala/GenerateService.groovy

```diff
@@ -204,6 +204,10 @@ class GenerateService {
                         taxon.imageDataResourceUrl = imageDataResourceMetadata.websiteUrl
                         taxon.imageDataResourceName = imageDataResourceMetadata.name
                     }
+                    if (imageMetadata.recognisedLicence != null) {
```

Comment:
Add resilience change to groovy test `if (imageMetadata?.recognisedLicence)`

Reply:
I had committed the changes.